pydra/engine/task.py

```diff
@@ -319,21 +342,28 @@ def command_args(self):
     def command_args(self, args: ty.Dict):
         self.inputs = attr.evolve(self.inputs, **args)
 
+    @property
+    def container_args(self):
```

Comment:
is this still required.

Reply:
I added this when I decided to remove `run_task` and `smndline` from `ContainerTask`. They were very similar

Reply:
but i can modify the methods and add an if statement